### ISSUE TITLE
Reproducibility issue in few-shot setting: performance gap with reported results

### ISSUE THREAD

**Issue by senquanwang** [NWPU-903PR/HyperSynergy#4]
Hi, thanks for releasing this excellent work and code!

I am trying to reproduce the few-shot results of HyperSynergy (Table 1), but I observe a significant gap even when strictly following the provided codebase.

**Paper (5-shot):**
- MSE: 0.115 ± 0.002
- SCC: 0.508 ± 0.004
- R2: 0.180 ± 0.008

**My results (5-shot):**
- MSE: 0.181
- SCC: 0.356
- **R2: -0.301**

<img width="500" height="300" alt="Image" src="https://github.com/user-attachments/assets/abae8ba3-0c29-4a50-bb2c-5c08b1490b68" />

**Setup:**
- Official dataset and provided split files
- Few-shot sampling strictly follows the repo implementation
- Evaluation protocol consistent with the paper
- **No modifications were made to the original code** except for changing the checkpoint used for testing in `test_few_shot.py`. Specifically, the original setting uses: `best_MSE, best_step = format(0.0648, '.4f'), 21600`. I replaced it with the best checkpoint obtained from my training process: `best_step = 37800, best_MSE = 0.0900`.

**Environment:**
- Python 3.10
- PyTorch 2.1.0 (CUDA 13.0)
- PyG 2.7.0

**Key observations:**
1. Training loss and validation loss are highly unstable and do not converge smoothly.

<img width="500" height="300" alt="Image" src="https://github.com/user-attachments/assets/3676602f-ec0c-4da5-8b72-2766a64d565b" />
<img width="500" height="300" alt="Image" src="https://github.com/user-attachments/assets/40d5df8b-b3ec-4cfb-ae58-288dfcbdcebb" />

2. Performance fluctuates significantly across runs.
3. In Table S5 (SI), a "Stopping step" is mentioned, but I could not find its implementation in the code. I also experimented with early stopping, but it did not significantly improve performance.

**Questions:**
Could you please clarify:
- whether any additional training schedule or stage-wise procedure is required for the few-shot setting?
- whether any implementation details (e.g., sampling strategy, training stages, or hyperparameters) are missing from the released code?

I would greatly appreciate any clarification. Thanks again!
